ocaml/xapi-cli-server/records.ml

```diff
@@ -168,10 +168,16 @@ let get_uuid_from_ref_or_null r =
 
 let concat_with_semi = String.concat "; "
 
+let concat_with_comma = String.concat ", "
```

Comment:
Let's add a comment to discourage new uses of this function, `map_and_concat_comma` and `get_uuids_from_refs_outdated` (explaining why).

Reply:
That was quick...

Reply:
;)